env.sh

```diff
@@ -3,10 +3,16 @@
 # source this file to get environment setup for the
 # pyang below here
 
-export PATH="$PWD/bin:$PATH"
 export MANPATH="$PWD/man:$MANPATH"
 export PYTHONPATH="$PWD:$PYTHONPATH"
 export YANG_MODPATH="$PWD/modules:$YANG_MODPATH"
 export PYANG_XSLT_DIR="$PWD/xslt"
 export PYANG_RNG_LIBDIR="$PWD/schema"
 export W="$PWD"
+
+if ! pyang --help &> /dev/null
```

Comment:
What exactly is this doing? If it's checking whether the `pyang` command already works then perhaps this could be done more directly? Perhaps it would be better to check this first, before changing `PYTHONPATH` etc., so the script either does nothing or does everything?